QualificationChecker/Controllers/UtilityClass.cs

```diff
@@ -0,0 +1,122 @@
+﻿using QualificationChecker.Models;
+
+namespace QualificationChecker.Controllers
+{
+    public abstract class UtilityClass
+    {
+        protected UtilityClass(Candidate candidate, Organization organization)
+        {
+            Candidate = candidate;
+            Organization = organization;
+        }
+
+        public Candidate Candidate { get; private set; }
+
+        public Organization Organization { get; private set; }
+
+        public Dictionary<int, List<OrgQuestion>> OrgAnswersDictionary
+        {
+            get
+            {
+                return Organization.OrgQuestions
+                                    .GroupBy(q => q.PositionId)
+                                    .ToDictionary(group => group.Key, group => group.ToList());
+            }
+        }
+
+    }
+    // fake incoming query call that an http request
+    public class MatchRequest
```

Comment:
Note: its generally considered a bad practice to include multiple classes in the same file (unless the classes are private)